static/js/chart/draw_line.ts

```diff
@@ -440,8 +445,11 @@ export function drawLineChart(
     });
     const hasGap = shouldFillInValues(dataset);
     hasFilledInValues = hasFilledInValues || hasGap;
+    // If there is a specific date to be highlighted on the chart, dont' show
+    // any other dots.
     const shouldAddDots =
-      dataset.length < MIN_POINTS_FOR_DOTS_ON_LINE_CHART || showAllDots;
+      !options.highlightDate &&
+      (dataset.length < MIN_POINTS_FOR_DOTS_ON_LINE_CHART || showAllDots);
```

Comment:
good point, 

- moved showAllDots to LineChartOptions
- if both showAllDots and highlightDate are set, then we still show the other dots and the highlighted one is just more prominent. If only highlight is set, we only show the highlighted dot. Went with this because".  
    - showAllDots only gets set as true for graph browser and graph browser wont be highlighting any points
    - highlighting dots when there are other dots shown actually still looks ok (https://screenshot.googleplex.com/36WVxaWevYNytww), just less obvious for certain colors (https://screenshot.googleplex.com/8u99pCkxkpcJmwV)

Reply:
This looks good!